relay-event-schema/src/protocol/span.rs

```diff
@@ -159,6 +159,14 @@ impl Getter for Span {
                     val.into()
                 } else if let Some(key) = path.strip_prefix("sentry_tags.") {
                     self.sentry_tags.value()?.get(key)?.as_str()?.into()
+                } else if let Some(key) = path.strip_prefix("measurements.") {
+                    self.measurements
+                        .value()?
+                        .get(key)?
+                        .value()?
+                        .value
+                        .value()?
+                        .into()
```

Comment:
Let's distinguish between `measurement.value` and `measurement.unit` here, as we do for events: https://github.com/getsentry/relay/blob/190ac90b72f9f938e3e3f7a5c94acbf659f7adfa/relay-event-schema/src/protocol/event.rs#L771-L773

https://github.com/getsentry/relay/pull/2938/files#r1450208498